woocommerce/woocommerce-android
# Add correct "stripe extension setup not completed" labels and illustration

Closes: #5331 


### Description

This PR takes care of displaying correct illustrations and labels when the Stripe Extension setup is not completed
Here is the Figma link for the designs: vaw3DvewbUwuQnPXpdNGGH-fi-2538%3A229187

### Testing instructions

1. Make sure the Stripe Extension plugin is installed but its setup is not completed **OR** add this line `return SetupNotCompleted(STRIPE_TERMINAL_GATEWAY)` in `CardReaderOnboardingChecker` line no: 57
2. Navigate to In-Person Payments via Settings
3. Make sure you see the correct illustrations and labels

### Images/gif

<img src=https://user-images.githubusercontent.com/1331230/146725442-2f4d848f-ad70-4ed1-9e19-526849a53dea.png width=250px/>

<img src=https://user-images.githubusercontent.com/1331230/146725524-a62dc141-2564-4a77-89bb-7f07b40a7364.png width=250px/>
